interferopy/casatools_vla_pipe.py

```diff
@@ -1,5 +1,5 @@
-"""VLA pipeline helper functions"""
-__author__ = "Mladen Novak"
+# """VLA pipeline helper functions"""
```

Comment:
```suggestion
"""VLA pipeline helper functions"""
```
I think you can leave this? Or just remove it altogether, since the other files also don't have a general description.

Reply:
It should be left as is or removed - the trip """ format then screws up the API.